system/jlib/jexcept.cpp

```diff
@@ -991,6 +991,7 @@ void excsighandler(int signum, siginfo_t *info, void *extra)
     signal(SIGBUS, SIG_DFL);
     signal(SIGILL, SIG_DFL);
     signal(SIGFPE, SIG_DFL);
+    signal(SIGABRT, SIG_DFL);
```

Comment:
By default a SIGABRT should generate a core file so I do not see a strong need to add this.